kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/JobServicesClientImpl.java

```diff
@@ -55,10 +55,14 @@ public Long scheduleRequest(String containerId, JobRequestInstance jobRequest) {
         if( config.isRest() ) {
 
             Map<String, Object> valuesMap = new HashMap<String, Object>();
-            valuesMap.put(CONTAINER_ID, containerId);
+            String queryString = "";
+            if (containerId != null && !containerId.isEmpty()) {
+                valuesMap.put(CONTAINER_ID, containerId);
```

Comment:
this is not needed, containerId does not have to be added to the values map sit is not a path param but query param only.